lib/Models/doesImageryFeatureBelongToCatalogItem.ts

```diff
@@ -0,0 +1,20 @@
+import MappableMixin, { ImageryParts } from "../ModelMixins/MappableMixin";
+import Feature from "./Feature";
+import { BaseModel } from "./Model";
+
+/**
+ * Returns true if the given feature belongs to the catalog item
+ */
+export default function doesImageryFeatureBelongToItem(
+  feature: Feature,
+  item: BaseModel
+): Boolean {
+  if (!MappableMixin.isMixedInto(item)) return false;
+  const imageryProvider = feature.imageryLayer?.imageryProvider;
+  if (imageryProvider === undefined) return false;
+  return (
+    item.mapItems.find(
+      m => ImageryParts.is(m) && m.imageryProvider === imageryProvider
+    ) !== undefined
```

Comment:
could use Array.includes here

Reply:
`Array.includes` only takes a concrete value, no matcher function, so I don't think we can use it here.

Reply:
ahh you are right, I was thinking of `Array.some` which takes in a matcher function

Reply:
Ah yes, that works. I have made the change :+1: